charms/openfga-k8s/tests/unit/test_charm.py

```diff
@@ -179,7 +153,20 @@ def test_on_openfga_relation_joined(
             "address": "10.10.0.17",
             "port": "8080",
             "scheme": "http",
-            "token": "test-token",
+            "token_secret_id": test_regex("secret:.*"),
             "store_id": "01GK13VYZK62Q1T0X55Q2BHYD6",
             "dns_name": "openfga-k8s-0.openfga-k8s-endpoints.None.svc.cluster.local",
         }
+
+
+class test_regex:
```

Comment:
Is this needed?

Reply:
what is the best way to assert string matches a regexp?

Reply:
Just curious if the class is needed, instead of just using the functions in this class directly.